core/services/offchainreporting/peerstore.go

```diff
@@ -73,7 +73,7 @@ func (p *Pstorewrapper) Start() error {
 
 func (p *Pstorewrapper) dbLoop() {
 	defer close(p.chDone)
-	ticker := time.NewTicker(p.writeInterval)
+	ticker := time.NewTicker(utils.WithJitter(p.writeInterval))
```

Comment:
This could really use one of you great comments explaining the why 🙂

Reply:
We use it in a few places around the codebase, there is a comment on the function definition if people are interested.